src/web/services/log.service.ts

```diff
@@ -20,7 +20,7 @@ export class LogService {
     courseId: string,
     feedbackSessionName: string,
     studentEmail: string,
-    logType: LogTypes }): Observable<string> {
+    logType: LogType }): Observable<string> {
     const paramMap: Record<string, string> = {
       courseid: queryParams.courseId,
       fsname: queryParams.feedbackSessionName,
```

Comment:
Since the endpoint was changed to post, the method for this method should change too

Reply:
👍